fpga/src/CMakeLists.txt

```diff
@@ -17,4 +17,9 @@ target_link_libraries(villas-fpga-pipe PUBLIC
 )
 
 
+add_executable(platform platform.cpp)
+target_link_libraries(platform PUBLIC
+	villas-fpga
```

Comment:
Isn't this just for your debugging? I think it's better to integrate the functionality in `villas-fpga-ctrl`